holoviews/annotators.py

```diff
@@ -142,12 +142,12 @@ def __call__(self, element, **params):
                            'method to combine them into a single layout.')
                     raise ValueError(msg)
                 annotator_type = sorted(matches)[0][1]
-                self.annotator = annotator_type(element, **params)
+                self.annotator = annotator_type(el, **params)
                 tables = Overlay([t[0].object for t in self.annotator.editor], group='Annotator')
                 layout = (self.annotator.plot + tables)
                 layers.append(layout)
             else:
-                layers.append(element)
+                layers.append(el)
 
         if annotator_type is None:
             obj = overlay if isinstance(overlay, Overlay) else element
```

Comment:
I think this should be the original element and not the element in the for loop.

Reply:
Don't think that's accurate, the change and original behavior seems correct. The intent is to either leave the layer as is or match it to an annotator.